api/src/main/resources/workbench-api.yaml

```diff
@@ -350,6 +350,33 @@ paths:
           description: Bad request
           schema:
             "$ref": "#/definitions/ErrorResponse"
+  "/v1/link-ras-account":
+    post:
+      tags:
+      - profile
```

Comment:
Is that the right Controller to use? feel it falls into profile category?

Reply:
Someday we may have an "AccessController" or something like that with the Access Module redesign.

But today: yes, Profile is correct